api/src/test/java/org/openmrs/api/PersonServiceTest.java

```diff
@@ -2187,4 +2244,216 @@ public void purgePersonAttributeType_shouldThrowAnErrorWhileTryingToDeletePerson
 		PersonAttributeType pat = ps.getPersonAttributeType(1);
 		ps.purgePersonAttributeType(pat);
 	}
+
+	//_____________________________________________________________________________________________________________________________-
+
+	/**
+	 * @verifies Return Null When Person Is Null
+	 * @see PersonServiceImpl#voidPerson(org.openmrs.Person, String)
+	 */
+	@org.junit.Test
```

Comment:
Removed, these were auto generated in the PersonServiceImplTest and I just copied them over as such.